canonical-websites/www.ubuntu.com
# 2685: Update /community page

# Note
- I have created the feature branch `build-community-section` for all community page related PRs, and will merge with `master` when it's finalised 

## Done

- Moved /about/about-ubuntu to new /community section and added redirects
- Updated Community copy according to [copy doc](https://docs.google.com/document/d/18sNzCqHj8Nf1w-JKRfnM9bvMaz2uZPPDgj_DISgJ30A/edit)
- Renamed /download/ubuntu-flavours.html to /download/flavours.html and added redirects
- Updated Ubuntu Flavours copy according to [copy doc](https://docs.google.com/document/d/1M5nkigcSRzq1YVjiXOZkeU4EUCqT0uYBR3Y_sjxM-wM/edit)
- Deleted /about/about-ubuntu/flavours.html

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/community
- Check that the Community content matches the [copy doc](https://docs.google.com/document/d/18sNzCqHj8Nf1w-JKRfnM9bvMaz2uZPPDgj_DISgJ30A/edit)
- Check that navigating to http://0.0.0.0:8001/about/about-ubuntu redirects to http://0.0.0.0:8001/community
- Check that the Ubuntu Flavours content matches the [copy doc](https://docs.google.com/document/d/1M5nkigcSRzq1YVjiXOZkeU4EUCqT0uYBR3Y_sjxM-wM/edit)
- Check that navigating to http://0.0.0.0:8001/download/ubuntu-flavours redirects to http://0.0.0.0:8001/download/flavours

## Issue / Card

Fixes #2684, fixes #2685
